calico_node/startup/startup.go

```diff
@@ -260,11 +260,14 @@ func determineNodeName() string {
 
 	// Determine the name of this node.  Precedence is:
 	// -  NODENAME
+	// -  Value stored in /var/lib/calico/nodename
 	// -  HOSTNAME (lowercase)
 	// -  os.Hostname (lowercase).
 	// We use the names.Hostname which lowercases and trims the name.
 	if nodeName = strings.TrimSpace(os.Getenv("NODENAME")); nodeName != "" {
 		log.Infof("Using NODENAME environment for node name")
+	} else if nodeName = nodenameFromFile(); nodeName != "" {
+		log.Info("Using stored value from /var/lib/calico/nodename")
```

Comment:
consider logging the `nodeName` as well.

Reply:
We log it out below, though seems fine to also log it here.